src/main/java/com/cebanova/classesandobjects/autoservicetask/ManageAutoService.java

```diff
@@ -0,0 +1,75 @@
+package com.cebanova.classesandobjects.autoservicetask;
+
+public class ManageAutoService {
+    public static void main(String[] args) {
+        // Declararea si initializarea obiectelor folosind toate tipurile de constructori
+
+        // Constructor cu parametri:
+        Garage AcoolaService = new Garage(20);
+        AcoolaService.address = "Alba Iulia 30";
+        AcoolaService.numberId = "2";
+        AcoolaService.workSchedule = "Luni-Vineri, 10:00-18:00";
+        AcoolaService.hasComputerDiagnostics = true;
+
+        System.out.println("AcoolaService are urmatoarele proprietati: Adresa: " + AcoolaService.address + "; Numarul: "
+                + AcoolaService.numberId + "; Programul de lucru: " + AcoolaService.workSchedule
+                + "; Diagnosticare Computerizata: " + AcoolaService.hasComputerDiagnostics + "; Capacitatea: "
+                + AcoolaService.carCapacity);
+
+        Garage HelpAuto = new Garage("Ion Creanga 20");
+        HelpAuto.numberId = "1";
+        HelpAuto.workSchedule = "Non stop";
+        HelpAuto.hasComputerDiagnostics = true;
+        HelpAuto.carCapacity = 10;
+
+        System.out.println("HelpAuto are urmatoarele proprietati: Adresa: " + HelpAuto.address + "; Numarul: "
+                + HelpAuto.numberId + "; Programul de lucru: " + HelpAuto.workSchedule
+                + "; Diagnosticare Computerizata: " + HelpAuto.hasComputerDiagnostics + "; Capacitatea: "
+                + HelpAuto.carCapacity);
+
+        Car Ferrari = new Car("Ferrari", "488 GTB", "black");
```

Comment:
Numele la variabila trebuie sa se inceapa cu lower case, respectiv trebuie sa fie ferrari in loc de Ferrari. Valabil pentru toate obiectele create.

Reply:
Done!

Mersi mult.